server.c

```diff
@@ -123,9 +123,22 @@ int main(void)
             if (status == 0) printf("Other connection closed");
             if (status == -1) perror("recv");
             printf("%s", str);
-            if (send(new_fd, "Hello, world!", 13, 0) == -1)
+
+	    // send file named index.html whenever we get a request
+	    FILE* home_pg = fopen("index.html", "r");
+	    fseek(home_pg, 0L, SEEK_END);
```

Comment:
This could be unsuccessful, returning a non-zero value.  If this happens we should return a 500 response code (internal server error, you can see all codes in the spec @ https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) to the client. Since the goal of this PR was just to get an end to end request/response of an html file, I think we can do that in a separate pull request because we need to implement a function (probably functions) to build a proper response (one that actually follows the HTTP spec: https://www.w3.org/Protocols/rfc2616/rfc2616.html) first.

In the meantime, perhaps you could just have this print text indicating that fseek failed with perror function (TODO: we need another PR for real logging... using syscall or something similar). We can just send something to the client ("ERROR" or something for now until we implement responses that meet spec) and then return since who knows would could happen if we continued past this error